microsite/data/plugins/buildkite.yaml

```diff
@@ -1,13 +1,10 @@
 ---
 title: Buildkite
-author: roadie.io
-authorUrl: https://roadie.io/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=buildkite
```

Comment:
@dtuite what's your preference on how to handle this one? Do you want to hand over to the new official one or keep yours around with a name that makes it clear that it's the Roadie edition?

Reply:
I'd imagine the official one will be better than our one so I'm fine with deprecating our plugin. What's the process for that? Do we just remove it from the marketplace or mark it as deprecated or what? If people are using it it might be weird if it just disappears completely.

Reply:
Marking it as deprecated on NPM would be the way to go. Afaik you can either do it via the CLI or publish a new version with `deprecated: true / <msg>` in `package.json`.